projects/css-custom-vars-viewer/package.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "@spectrum-web-components/custom-vars-viewer",
+    "version": "0.0.0",
+    "private": "true",
+    "description": "Demo app created with Spectrum Web Components, following open-wc guidelines",
+    "license": "Apache-2.0",
+    "author": "najikahalsema",
+    "main": "dist/src/index.js",
+    "module": "dist/src/index.js",
+    "type": "module",
+    "exports": {
+        ".": "./dist/src/index.js",
+        "./sp-css-table.js": "./dist/src/sp-css-table.js",
+        "./custom-vars-viewer.js": "./dist/src/custom-vars-viewer.js"
+    },
+    "engines": {
+        "node": ">=16.14.2",
+        "yarn": ">=1.16.0"
+    },
+    "scripts": {
+        "analyze": "cem analyze --litelement",
+        "build": "tsc && yarn parse-json && npm run analyze -- --exclude dist",
```

```suggestion
        "build": "yarn parse-json && tsc && yarn parse-json && npm run analyze -- --exclude dist",
```

I _think_ this is what is blocking the build...but, I've not been right in many tries so far 😬